### PR TITLE
Build updates

### PR DESCRIPTION
**Pull request by orien**
* Test the combination of Rails 3.2 and Ruby 2.2 (requires the `test-unit` gem).
* Update to new TravisCI agents

### CODE REVIEW

File: double_entry.gemspec [hunk 1]
```diff
@@ -62,6 +62,7 @@ POSTINSTALLMESSAGE
   gem.add_development_dependency 'generator_spec'
   gem.add_development_dependency 'machinist'
   gem.add_development_dependency 'timecop'
+  gem.add_development_dependency 'test-unit'
```
Comment (orien) [inline double_entry.gemspec]: Ruby 2.2 doesn't supply `test-unit` while the previous versions do. Rails 3.2 requires this.

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,11 @@
 language: ruby
+sudo: false
```
Comment (orien) [inline .travis.yml]: This allows us onto the new TravisCI agent infrastructure.

**Comment by macosgrove** [comment]
:+1:

**Comment by joneslee85** [comment]
:+1:

**Pull request merged**